cime_config/buildnml

```diff
@@ -340,13 +350,22 @@ def buildnml(case, caseroot, compname):
         if run_type == "hybrid" or run_type == "branch":
             compnames = ["clm4", "clm5", "clm2"]
             for comp in compnames:
-                clm_startfile = "%s.%s%s.r.%s-%s.nc" % (
-                    run_refcase,
-                    comp,
-                    inst_string,
-                    run_refdate,
-                    run_reftod,
-                )
+                if "PLUMBER2" in clm_usrdat_name:
+                    clm_startfile = "%s.%s%s.r.%s-%s.nc" % (
+                        run_refcase,
+                        comp,
+                        inst_string,
+                        run_refdate,
+                        start_tod,
+                    )
+                else:
+                    clm_startfile = "%s.%s%s.r.%s-%s.nc" % (
+                        run_refcase,
+                        comp,
+                        inst_string,
+                        run_refdate,
+                        run_reftod,
```

Comment:
OK, the difference here is whether start_tod or run_reftod is being given. This could use a comment or a restructuring to make that obvious. Having something like

``` python
if "PLUMBER2" in clm_usrdat_name:
   tod = start_tod
else:
   tod = run_reftod

   clm_startfile = "%s.%s%s.r.%s-%s.nc" % (
                        run_refcase,
                        comp,
                        inst_string,
                        run_refdate,
                        tod,
   )
```

makes it obvious what the difference is for PLUMBER2.

Reply:
That is much clearer! Thanks for the suggestion.